website/docusaurus.config.js

```diff
@@ -12,7 +12,8 @@ const config = {
     baseUrl: '/flutter_rust_bridge/',
     onBrokenLinks: 'throw',
     onBrokenMarkdownLinks: 'warn',
-    favicon: 'img/favicon.ico', // TODO
+    favicon: 'favicon.ico',
+    staticDirectories: ['misc'],
```

Comment:
nit: I personally suggest adding `public` as a static dir (instead of the misc - which contains a lot of other things). Then, just `ln -s` (make a symbolic link) from the real png